system/ad_service_state_monitor/src/ad_service_state_monitor_node/ad_service_state_monitor_node.cpp

```diff
@@ -122,6 +122,11 @@ void AutowareStateMonitorNode::onOdometry(const nav_msgs::msg::Odometry::ConstSh
 
     state_input_.odometry_buffer.pop_front();
   }
+
+  constexpr size_t odometry_buffer_size = 40;
```

Comment:
I think it's good to have some margin and add a comment.

```suggestion
  constexpr size_t odometry_buffer_size = 200; // 40Hz * 5sec
```